### PR TITLE
Translate of blog.rst to pt_BR

### PR DESCRIPTION
**Pull request by viniciusls**
This is my translation of Blog Tutorial - CakePHP 3.0 from English to Portuguese (Brazil).
I hope it helps!

Anyway, congratulations by your framework. It helps me a lot :)
Att. Vinícius Luiz

### CODE REVIEW

File: pt/tutorials-and-examples/blog/blog.rst [hunk 1]
```diff
@@ -1,13 +1,132 @@
...
-.. note::
-    A documentação não é atualmente suportada pela lingua portuguesa nesta
-    página.
+•	Criando um blog 
+•	Instalação do CakePHP
+•	Permissões dos diretórios tmp e logs
+•	Criando o Banco de Dados do blog
+•	Configuração do Banco de Dados
+•	Configuração opcional
+•	Observação sobre o mod_rewrite
```
Comment (markstory) [inline pt/tutorials-and-examples/blog/blog.rst]: The table of contents is automatically generated by the documentation tool.

File: pt/tutorials-and-examples/blog/blog.rst [hunk 2]
```diff
@@ -1,13 +1,132 @@
-Blog Tutorial
-#############
+Conteúdos da página
```
Comment (markstory) [inline pt/tutorials-and-examples/blog/blog.rst]: Headings need underline markings.

**Comment by markstory** [comment]
Looks like most of the formatting has been lost in the process. I can re-add the formatting before merging this.

**Comment by viniciusls** [comment]
Yeah, I did just the translate in a Word Doc and paste here. It's the first time that I did it, so sorry me about this kind of mistake. Anyway, I hope that the translate helps ^.^

**Comment by markstory** [comment]
Thanks for doing the translation, we can work on the formatting in future pull request. :smile:

**Comment by viniciusls** [comment]
You're welcome. I want to continue translating the docs. Can I continue doing this way with the other parts of the Blog Tutorial?
Or do you have some tips to me about the formatting?
Is there a prediction of how long it will take to append it to official docs if that's okay? I'm excited in helping.

**Comment by markstory** [comment]
If you're going to do a bunch more translations it would be simpler if you kept the existing sphinx formatting around. The [documentation](http://book.cakephp.org/2.0/en/contributing/documentation.html) has a quick overview on RST.

As for time to incorporate into the live docs, I hope to have this merged and deployed tonight.

**Comment by viniciusls** [comment]
Alright, I'll try to follow all those tips in future translations so :D

**Pull request merged**